### PR TITLE
fix: add `original-wordmark` aliases for Liquibase

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add `original-wordmark` aliases for Liquibase.
  - Thanks to @kilian-paquier for spotting the bug!

### CODE REVIEW

**Comment by kilian-paquier** [comment]
> > @Panquesito7 Isn't there a missing `}, {` for the second alias ?
> 
> 
> 
> Good catch! Thank you. 🙂
> 
> 
> 
> 
> 
> 

No problem 🙂 !

**Review by kilian-paquier — APPROVED** [review]

File: devicon.json [hunk 1]
```diff
@@ -7193,6 +7193,11 @@
             {
                 "base": "original",
                 "alias": "plain"
+            },
+
```
Comment (Snailedlt) [inline devicon.json:7197]: unnessesary whitespace here :)
```suggestion
```

**Review by Snailedlt — COMMENTED** [review]
Other than an extra whitespace it looks good ! :D

**Review by Snailedlt — APPROVED** [review]
LGTM!

**Review by kilian-paquier — APPROVED** [review]

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**